src/main/java/org/spongepowered/api/Server.java

```diff
@@ -274,7 +274,7 @@
 
     /**
      * Shuts down the server, and kicks all players with the default kick
-     * message (the translation key {@code disconnect.closed}).
+     * message (the literal text "Server closed").
```

Comment:
I realized that what I thought was the kick message was actually not, so I just slipped this change in here. Perhaps it should not be defined in the docs.